webhookbot/main.go

```diff
@@ -48,7 +48,30 @@ func (s *BotServer) makeAdvertisement() kbchat.Advertisement {
 	createExtended := fmt.Sprintf(`Create a new webhook for sending messages into the current conversation. You must supply a name as well to identify the webhook. To use a webhook URL, supply a %smsg%s URL parameter, or a JSON POST body with a field %smsg%s.
 
 	Example:%s
-		!webhook create alerts%s`, back, back, back, back, backs, backs)
+		!webhook create alerts%s
+
+	Example (using custom template):%s
+		!webhook create alerts *{{.title}}*
+		%s{{.body}}%s%s
+
+	Templates:
+		Information about creating templates, and some examples can be found at %shttps://pkg.go.dev/text/template%s
```

Comment:
i forget, are these links clickable from keybase chat?

Reply:
just double checked and no, they stay as plaintext in the command ad

Reply:
maybe there should be a `!help` command that prints this then?